rllib/BUILD

```diff
@@ -249,6 +249,16 @@ py_test(
     args = ["--yaml-dir=tuned_examples/ppo"]
 )
 
+py_test(
```

Comment:
This task is now working properly. Due to proper hyperparam tuning.

Reply:
It was pretty hard, actually.
It's good to start with just one worker and using the exact same hparams as the respective PPO version. Then increase num_workers and at the same time carefully adjust:
```
rollout_fragment_length
num_envs_per_worker
sgd_minibatch_size
num_sgd_iter
```

in short, anything that affects (per-worker) batch-size, and the time each worker spends on a decentralized update.